package.json

```diff
@@ -11,8 +11,8 @@
     "lint": "eslint .",
     "e2e": "ng e2e",
     "docs:json": "compodoc -p ./tsconfig.json -e json -d .",
-    "storybook": "npm run docs:json && start-storybook -p 6006",
-    "build-storybook": "npm run docs:json && build-storybook",
+    "storybook": "npm run docs:json && start-storybook -p 6006 -s projects/ion/src",
```

Comment:
`-s` flag is deprecated, you'll have to add a config file. Check the doc.
https://storybook.js.org/docs/react/api/cli-options#static-dir-deprecation